### ISSUE TITLE
[BITV]: Replace custom tooltips with native one for personal and administration settings

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#35886]
## Problem

To show a tooltip on a several components we currently use v-tooltip. An example can be found in https://github.com/nextcloud/server/blob/master/apps/settings/src/components/PersonalInfo/shared/FederationControl.vue#L37

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to several components that will render as `title` attribute on the HTML element.
